TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config

```diff
@@ -155,26 +155,6 @@ train_config: {
   # effectively bypasses the learning rate schedule (the learning rate will
   # never decay). Remove the below line to train indefinitely.
   num_steps: 15000
```

Comment:
In my opinion, 15k are much higher, because it would take a lot of time.
Shall we make it 10k? We already have to do around 10+ variations

Reply:
ok, 10k gave me same loss. maybe at least 12k, because of on loss plot 10k it seems like 10k not fully explores loss